src/assets/VueOptionVsCompositionAPI/option-vs-composition-api.md

```diff
@@ -0,0 +1,186 @@
+---
+title: Option Vs Composition API in Vue JS
+authorName: Sagar Gurung
+authorAvatar: https://avatars.githubusercontent.com/u/46086950?v=4
+authorLink: https://github.com/SagarGi
+createdAt: Dec 13, 2023
+tags: vue, option API, composition API, framework, frontend
```

Comment:
Vue (or Vue.js), Options API and Composition API are usually written with capital letter because they are names. Try to be consistent through the blog about how you write these terms, also regarding using quotation marks for these terms or not...